### DISCUSSION TITLE
Getting 403 trying to edit an Entry

### DISCUSSION THREAD

**Discussion by ribeiro-jr** [statamic/cms#4742] [Troubleshooting]
Hi,

After uploading the project into production I can only create and delete data entry. But when I want to edit I get 403, as you can see in the image below.

![image](https://user-images.githubusercontent.com/63875039/141995375-9de12f61-7116-43a4-b11b-070bb858647e.png)

I need your help.

Note: I'm using Statamic v3 on a shared hosting.

**Comment by sebszocinski** [comment]
@ribeiro-jr I'd check your file permissions, sounds like there could be an issue there. Are you able to view your server log files? they would give you more info. I'm guessing you might be using cPanel? See here: https://docs.cpanel.net/knowledge-base/cpanel-product/the-cpanel-log-files/

**Reply by ribeiro-jr** [reply]
@sebszocinski I tried to set 777 permission to content folder but nothing happen. So I took a look at log files and I find something inside of it matching with the datetime I made the request:

`[msg "Method is not allowed by policy"] [data "PATCH"]`

Unfortunately I don't have access to my hosting conf files, so I contacted my hosting provider in order for them to allow **PATCH** request, and they might respond until tomorrow.

Thanks. ;)

**Reply by sebszocinski** [reply]
@ribeiro-jr yeah ok that looks like a server/hosting issue. Fingers crossed they can solve it for you.
